plugin-modernizer-core/src/test/resources/plugins.txt

```diff
@@ -0,0 +1,3 @@
+jobcacher
```

Comment:
Did we try with a plugin file that contains version information ? Are we spliting the list correctly ?

Reply:
Here is one I use on a daily basis: https://github.com/jenkins-docs/quickstart-tutorials/blob/main/dockerfiles/plugins.txt .